client/src/Components/UserDashboard/UserDashboard.module.css

```
@@ -0,0 +1,34 @@
+.imgContainer{
+	width: 100px;
+	height: 100px;
+	border-radius: 50%;
+	background-position: center center;
+	background-size: cover;
+	position: relative;
+}
+
+.editIcon{
+	position: absolute; 
+	bottom:90%; 
+	left:0;
+	color: #ffffff;
+}
```

Please, pay attention to these details. I'd suggest using the pre-commit hook with eslint and typescript checks.

```suggestion
.editIcon {
	position: absolute; 
	bottom: 90%; 
	left: 0;
	color: #ffffff;
}
```